packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/upgrade/translations/Messages_fr_FR.json

```
@@ -3,11 +3,11 @@
   "dedicatedCloud_datacenter_backup_upgrade_confirm": "Passer à l’offre {{ name }}",
   "dedicatedCloud_datacenter_backup_upgrade_cancel": "Annuler",
   "dedicatedCloud_datacenter_backup_upgrade_description": "Vous souhaitez changer votre offre de sauvegarde de l’<strong>offre {{ currentOffer }}</strong> à l’<strong>offre {{ selectedOffer }}</strong>.",
-  "dedicatedCloud_datacenter_backup_upgrade_description_gain_features": "Vous obtiendrez les options suivantes :",
-  "dedicatedCloud_datacenter_backup_upgrade_description_lose_features": "Vous perdrez les options suivantes :",
+  "dedicatedCloud_datacenter_backup_upgrade_description_features": "Vous aurez désormais les paramètres suivants :",
   "dedicatedCloud_datacenter_backup_upgrade_description_confirmation": "Êtes-vous sûr(e) de vouloir changer d’offre ?",
   "dedicatedCloud_datacenter_backup_upgrade_description_billing": "Vous serez facturé(e) selon le nombre et la taille des machines virtuelles sauvegardées.",
-  "dedicatedCloud_datacenter_backup_upgrade_success_1": "Votre offre de sauvegarde a bien été activée. Vous êtes maintenant abonné à l'offre {{offer}}.",
-  "dedicatedCloud_datacenter_backup_upgrade_success_2": "Si vous souhaitez à nouveau Offre de sauvegarde, merci d’attendre 15 jours.",
+  "dedicatedCloud_datacenter_backup_upgrade_replication_condition": "Je valide la réplication des données",
+  "dedicatedCloud_datacenter_backup_upgrade_success_1": "La demande de changement de votre offre Veeam {{actualOffer}} à {{newOffer}} est en cours et prendra un peu de temps.",
+  "dedicatedCloud_datacenter_backup_upgrade_success_2": "Vous pouvez suivre l'avancement de l'opération <a href='{{operationsUrl}}'>ici</a>.",
```

FYI, you can do:
```suggestion
  "dedicatedCloud_datacenter_backup_upgrade_success_2": "Vous pouvez suivre l'avancement de l'opération <a href=\"{{operationsUrl}}\">ici</a>.",
```
;)

---

This is done. Thanks for the suggestion :)